Tests/AK/TestTime.cpp

```diff
@@ -690,12 +695,19 @@ TEST_CASE(time_to_string)
 TEST_CASE(parse_time)
 {
     auto test = [](auto format, auto time, int year, int month, int day, int hour, int minute, int second = 0) {
+        printf("Testing format: %s with time: %s\n", format.to_byte_string().characters(), time.to_byte_string().characters());
```

Comment:
Instead of `printf`, you should use `dbgln_if` here to conditionally toggle some debug messages.